documentation/docs/80-adapter-api.md

```diff
@@ -26,7 +26,6 @@ The types for `Adapter` and its parameters are available in [types/config.d.ts](
 Within the `adapt` method, there are a number of things that an adapter should do:
 
 - Clear out the build directory
-- Call `utils.update_ignores` to ignore build output in existing `.gitignore` files at the location of `svelte.config.js`
```

Comment:
It may be worth mentioning somewhere in here that we recommend putting the adapter output under `'.svelte-kit/' + adapterName`